lms/envs/common.py

```diff
@@ -1919,6 +1919,9 @@
     'teams',
 
     'xblock_django',
+
+    # edX Proctoring
+    'edx_proctoring',
```

Comment:
Note to reviewers, while this PR is just to enable proctoring in Studio, this is required to get tests in openedx/core/djangoapps/credit to pass (I presume they are running under a LMS configuration)